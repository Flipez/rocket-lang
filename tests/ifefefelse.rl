i = 0
while (i<4)
  if (i==0)
    puts("if/ef/el: if block")
  ef (i==1)
    puts("if/ef/ef/el: first ef block")
  ef (i==2)
    puts("if/ef/ef/el: second ef block")
  else
    puts("if/ef/else: else block")
  end

  i = i+1
end
