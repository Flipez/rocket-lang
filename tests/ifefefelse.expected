"if/ef/el: if block"
"if/ef/ef/el: first ef block"
"if/ef/ef/el: second ef block"
"if/ef/else: else block"
nil
